torchgeo/datasets/utils.py

```diff
@@ -598,6 +599,26 @@ def unbind_samples(sample: Dict[Any, Sequence[Any]]) -> List[Dict[Any, Any]]:
     return _dict_list_to_list_dict(sample)
 
 
+def flatten_samples(samples: Iterable[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
This looks a lot like `collate_wrapper` in the inria and oscd datamodules. Can you replace those with imports of this?